src/content/nl/2020/security.md

```diff
@@ -777,7 +777,7 @@ Een ander mechanisme om de adoptie van CSP gemakkelijker te maken, is het gebrui
 
 De twee andere trefwoorden, `unsafe-inline` en `unsafe-eval`, zijn aanwezig in de meeste CSP's: respectievelijk 97,28% en 77,79%. Dit kan worden gezien als een herinnering aan de moeilijkheid om een beleid te implementeren dat XSS-aanvallen kan dwarsbomen. Als het trefwoord `strict-dynamic` echter aanwezig is, worden de trefwoorden `unsafe-inline` en `unsafe-eval` effectief genegeerd. Omdat het trefwoord `strict-dynamic` mogelijk niet wordt ondersteund door oudere browsers, wordt het als een best practice beschouwd om de twee andere onveilige trefwoorden op te nemen om de compatibiliteit voor alle browserversies te behouden.
 
-Terwijl de trefwoorden `strict-dynamic` en `nonce-` kunnen worden gebruikt ter verdediging tegen gereflecteerde en aanhoudende XSS-aanvallen, kan een beschermde pagina nog steeds kwetsbaar zijn voor op DOM gebaseerde XSS-kwetsbaarheden. Om zich te verdedigen tegen deze klasse van aanvallen, kunnen website-ontwikkelaars gebruik maken van <a hreflang="en" href="https://web.dev/trusted-types/">Trusted Types</a>, een vrij nieuw mechanisme dat momenteel alleen wordt ondersteund door op Chromium gebaseerde browsers. Ondanks de mogelijke problemen bij het adopteren van vertrouwde typen (websites zouden een beleid moeten opstellen en mogelijk hun JavaScript-code moeten aanpassen om aan dit beleid te voldoen), en gezien het feit dat het een nieuw mechanisme is, is het bemoedigend dat 11 homepagina's al vertrouwde typen hebben aangenomen via de richtlijn `require-trusted-types-for` in CSP.
+Terwijl de trefwoorden `strict-dynamic` en `nonce-` kunnen worden gebruikt ter verdediging tegen gereflecteerde en aanhoudende XSS-aanvallen, kan een beschermde pagina nog steeds kwetsbaar zijn voor op DOM gebaseerde XSS-kwetsbaarheden. Om zich te verdedigen tegen deze klasse van aanvallen, kunnen website-ontwikkelaars gebruik maken van <a hreflang="en" href="https://web.dev/trusted-types/">Trusted Types</a>, een vrij nieuw mechanisme dat momenteel alleen wordt ondersteund door op Chromium gebaseerde browsers. Ondanks de mogelijke problemen bij het adopteren van vertrouwde typen (websites zouden een beleid moeten opstellen en mogelijk hun JavaScript-code moeten aanpassen om aan dit beleid te voldoen), en gezien het feit dat het een nieuw mechanisme is, is het bemoedigend dat 11 startpagina's al vertrouwde typen hebben aangenomen via de richtlijn `require-trusted-types-for` in CSP.
 
 ### Bescherming tegen XS-Leaks met Cross-Origin-Policies
```

Comment:
Shouldn't be a dash before the word `Policies` as this refers to a set of Cross-Origin policies: COOP (Cross-Origin-Opener-Policy), CORP (Cross-Origin-Resource-Policy), and COEP (Cross-Origin-Embedder-Policy).

```suggestion
### Bescherming tegen XS-leaks met Cross-Origin policies
```

Since we noticed this here, Any chance you could add English to this PR to change capitalisation to "Defending against XS-leaks with Cross-Origin policies"?